src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/EnvelopeProcessor.java

```diff
@@ -147,6 +151,29 @@ public static void assertEnvelopeHasPdfs(InputEnvelope envelope, List<Pdf> pdfs)
         }
     }
 
+    /**
+     * Assert envelope contains at least one scannable item
+     * with valid document type and ocr data for new application.
+     * Throws exception otherwise.
+     *
+     * @param envelope to assert against
+     */
+    public static void assertEnvelopeContainsOcrDataForNewApplication(InputEnvelope envelope) {
```

Comment:
Currently, we've got a mix of static and non-static `assert...` methods. It would be best if all of them were the same. My preference would be to make them all non-static, as it's easier to test code with those. What do you think?

Reply:
I think non-static is better